changelog.json

```
@@ -1,5 +1,17 @@
 {
   "releases": {
+    "3.1.3-beta4": [
+      "[Fixed] Hide window instead of hide the app in macOS - #15511. Thanks @angusdev!",
```

```suggestion
      "[Fixed] Hide window instead of hiding the app on macOS - #15511. Thanks @angusdev!",
```

(@tidy-dev as native speaker, I'll let you decide what's grammatically right 😂 )

---

I have heard it both ways.. so I went to go find out if there was a grammatically correct way. :D https://ell.stackexchange.com/questions/72067/which-is-the-right-preposition-when-referring-to-os-in-or-on

Seems not really but on is more common. :)